### PR TITLE
fixed hidden LO submition button bug

### PR DESCRIPTION
**Pull request by AStoerkel1**
this PR is related to [8718](https://app.shortcut.com/clarkcan/story/8718/clark-submission-button-size-issue) in shortcut

### CODE REVIEW

File: src/app/onion/shared/submit/submit.component.scss [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 .submit {
   max-width: 800px;
-  min-height: 550px;
+  height: 85vh;
```
Comment (EVogel1999) [inline src/app/onion/shared/submit/submit.component.scss:5]: I think we should use a mix of ```max-height``` and ```height``` here instead.  That way the popup isn't too big on larger screens.

**Review by EVogel1999 — COMMENTED** [review]
One suggestion.

File: src/app/onion/shared/submit/submit.component.scss (hunk 1 above)
Comment (AStoerkel1) [reply src/app/onion/shared/submit/submit.component.scss:5]: I can give it a shot but the popup doesn't really do what you would expect it to do, even set to 85vh it is still kinda cut off on the top and bottom when zoomed in far enough to experience the bug.

Comment (AStoerkel1) [reply src/app/onion/shared/submit/submit.component.scss:5]: ok I added a max-height of 550px

**Review by EVogel1999 — APPROVED** [review]
Looks good!  Thank you!

**Review by pzalep1 — APPROVED** [review]

**Pull request merged**